buffers.go

```diff
@@ -0,0 +1,24 @@
+package tlsproxy
```

Comment:
You could just import this from lampshade the way that flashlight does.

Reply:
Or we could move that into https://github.com/getlantern/common

Reply:
Do you think either of those are necessary?  Seems odd to have this project rely on lampshade for essentially a couple of constants.  Also no other projects were using the `http-proxy/buffers` package.

Reply:
FWIW I found `github.com/libp2p/go-buffer-pool` neat when skimming over the Geph code and used it in the `multipath` package. It allocates buffers with size in power of 2 and large enough to hold requested size, then returns a slice of the buffer exactly the size requested. It is less memory efficient if most frames are approaching `maxFrameSize`, but should save memory if many frames are small. The main benefit to me is to avoid using helper files like this one.

Reply:
Okay, I've switched to that library.  Thanks @joesis!